src/current/v24.1/disaster-recovery-overview.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Disaster Recovery Overview
+summary: Learn how to make your cluster resilient with high availability and disaster recovery features.
+toc: true
+---
+
+Resilient deployments aim for continuity in database operation to protect from data loss and down time. To maintain resiliency, it is necessary to build deployments with _high availability_ and _disaster recovery_ coverage.
+
+- [High availability](#choose-a-high-availability-strategy): Continuous and uninterrupted access to data even in the presence of failures or disruptions to maximize uptime.
+- [Disaster recovery](#choose-a-disaster-recovery-strategy): Recovery from a major incident or disaster to minimize downtime and data loss.
+
+To build resilient cluster deployments, CockroachDB provides both [built-in replication]({% link {{ page.version.version }}/architecture/replication-layer.md %}) for high availability and disaster recovery features for coverage during unplanned incidents.
+
+{{site.data.alerts.callout_success}}
+For a practical guide on how CockroachDB replicates, distributes, and rebalances data, refer to the [Replication and Rebalancing]({% link {{ page.version.version }}/demo-replication-and-rebalancing.md %}) demo.
+{{site.data.alerts.end}}
+
+## Resilience strategy
+
+As you evaluate CockroachDB's disaster recovery features, consider your organization's requirements for the amount of tolerable data loss and the acceptable length of time to recover.
+
+- Recovery Point Objective (RPO): The maximum amount of time that an organization can tolerate losing data.
+- Recovery Time Objective (RTO): The maximum length of time it should take to restore normal operations following an outage.
+
+For example, when you use backups:
+
+<img src="{{ 'images/v24.1/rpo-rto.png' | relative_url }}" alt="Simulating RPO and RTO. With RPO representing the tolerable data loss, and RTO representing the tolerable time to recovery." style="border:0px solid #eee;max-width:100%" />
+
+For a comparison of CockroachDB resiliency features, refer to the following sections:
+
+- [Choose a high availability strategy](#choose-a-high-availability-strategy)
+- [Choose a disaster recovery strategy](#choose-a-disaster-recovery-strategy)
+
+### Choose a high availability strategy
+
+CockroachDB uses synchronous, built-in replication to create and distribute copies of data, ensuring consistency across the data copies. The database can tolerate nodes going offline without service interruption whether in a single-region or [multi-region]({% link {{ page.version.version }}/multiregion-overview.md %}) cluster.
+
+<table class="comparison-chart">
+  <tr>
+    <th></th>
+    <th>Single-region replication</th>
```

Comment:
Is it worth adding "synchronous" replication here?

Reply:
Added to both columns here.